rpc/lib/client/ws_client_test.go

```diff
@@ -191,3 +194,55 @@ func callWgDoneOnResult(t *testing.T, c *WSClient, wg *sync.WaitGroup) {
 		}
 	}
 }
+
+func TestWSClientReconnectWithJitter(t *testing.T) {
```

Comment:
good that you wrote a test, but don't think we needed to test that specifically

Reply:
Agree that this might not be the most useful test, especially given how much code it is. We'd also have to start using `-skip`, and let's not.

That said, I like the idea of adding more advanced tests like this, but let's keep them separate from the unit testing. Perhaps we can come up with a good schema for starting to add all these longer-running, more nuanced tests, including the fuzzer stuff. We can run those tests on release, but they don't need to run every time with the unit tests.

Reply:
You can use a build tag like 'integration' (e.g. ithttps://github.com/hyperledger/burrow/blob/master/rpc/tendermint/test/rpc_client_test.go#L1-L2 - note you need the blank line after - run as https://github.com/hyperledger/burrow/blob/master/Makefile#L129)

Reply:
`good that you wrote a test, but don't think we needed to test that specifically`
`Agree that this might not be the most useful test, especially given how much code it is. We'd also have to start using -skip, and let's not.`
See the problem here is that that code can be changed at anytime and without a comprehensive test, we'd never know what changed at all. Perhaps we need some middle ground, like @ebuchman mentioned, perhaps as a release test.

Great suggestion @silasdavis, thank you.